test/metabase/db/schema_migrations_test.clj

```diff
@@ -524,7 +525,7 @@
           (testing (str "View " view-name " should be created")
             ;; Just assert that something was returned by the query and no exception was thrown
             (is (partial= [] (t2/query (str "SELECT 1 FROM " view-name))))))
-        #_#_ ;; TODO: this is commented out temporarily because it flakes for MySQL (metabase#37434)
+        #_#_ ;; TODO: this is commented out temporarily because it flakes for MySQL (metabase#37884)
```

Comment:
I updated the issue number because the newer one has more context about the decision to comment out these MySQL flakes

Reply:
wow, this one must really bugs people; it gets commented out twice.

Reply:
The first comment is for the `migrate!` form and the next one is for the `testing` form. It's neat that you can stack them like this.

Reply:
ohh. TIL!